src/index.html

```diff
@@ -11,7 +11,181 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="input_block">
+          Surname:
+          <input
+            class="input"
+            name="surname"
+            type="text"
+            id="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="input_block">
+          Name:
+          <input
+            class="input"
+            name="name"
+            type="text"
+            id="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="input_block">
+          How old are You?
+          <input
+            class="input"
+            name="age"
+            type="number"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="input_block">
+          Full date of birth:
+          <input
+            class="input"
+            name="birth"
+            type="date"
+            id="birth"
+          >
+        </label>
+        <label class="input_block">
+          I accept the term of the agreement
+          <input
+            class="input"
+            name="agreement"
+            type="checkbox"
+            id="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label class="input_block">
+          E-mail:
+          <input
+            class="input"
+            name="mail"
+            type="email"
+            id="mail"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="input_block">
+          Password:
+          <input
+            class="input"
+            name="pwd"
+            type="password"
+            id="pwd"
+            minlength="8"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="input_block">
+          Do you love cats?
+          <label>
+            <input
+              name="cats"
+              type="radio"
+              id="cats_yes"
```

Comment:
are you sure you need this id?

Reply:
I forgot to delete. It was there for the label